pkg/plugin/predicates/predicate_manager_test.go

```diff
@@ -1094,6 +1094,11 @@ func makeResources(milliCPU, memory, pods, extendedA, storage, hugePageA int64)
 func newPodWithPort(hostPorts ...int) *v1.Pod {
 	var networkPorts []v1.ContainerPort
 	for _, port := range hostPorts {
+		// Check for integer overflow before conversion
+		if port <= 0 || port > 65536 {
+			log.Log(log.Test).Error("invalid port number", zap.Int("port", port))
+			continue
+		}
```

Comment:
This is test code, no need to be this vigilant. Use `nolint` to suppress the linter.

Reply:
Yeah, make sense!